101-managed-application/README.md

```diff
@@ -0,0 +1,84 @@
+# Deploying your first Managed Application with default view
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-managed-application%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-managed-application%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This sample template deploys a Service catalog managed application definition that creates a single storage account as an application resource.  The application definition demonstrates how you can customize default application overview: header and description.
+
+[To learn more about managed applications click here](https://docs.microsoft.com/en-us/azure/managed-applications/overview)
+
+This template deploys a Service catalog managed application definition first, after that uses this application definition to  deploy a managed application.
+You can also use the package located under the artifacts folder to deploy the managed application as Azure Marketplace application offer.
+
+To learn more about Managed Application definitions and Marketplace please visit :
+
++ [Publish an Azure managed application definition](https://docs.microsoft.com/en-us/azure/managed-applications/publish-managed-app-definition-quickstart)
++ [Azure managed applications in the Marketplace](https://docs.microsoft.com/en-us/azure/managed-applications/publish-marketplace-app)
+
+## Exploring created Managed Application
+
+This sample deployment creates the following two resources:
```

Comment:
can we add something here that explains this is not the way you would normally do it... the definition is deployed separately from the app... otherwise I think we risk setting the impression that they must be deployed together.